CoMD/src-threaded/CoMDTypes.h

```diff
@@ -106,21 +106,48 @@ typedef RAJA::KernelPolicy<
   RAJA::statement::For<1, RAJA::seq_exec,
   RAJA::statement::For<2, RAJA::seq_exec,
   RAJA::statement::For<3, RAJA::simd_exec,
-  RAJA::statement::Lambda<0> > > > > > ljForcePolicy;
+  RAJA::statement::Lambda<0> > > > > > forcePolicy;
 
 typedef RAJA::ReduceSum<RAJA::omp_reduce, real_t> rajaReduceSumReal;
+
+#elif DO_CUDA
+
+typedef RAJA::seq_segit linkCellTraversal;
+typedef RAJA::ExecPolicy<RAJA::seq_segit, RAJA::simd_exec> linkCellWork;
+typedef RAJA::ExecPolicy<RAJA::seq_segit, RAJA::simd_exec> atomWork;
+typedef RAJA::seq_segit task_graph_policy;
+
+typedef RAJA::KernelPolicy<
```

Comment:
Here is where you should have the ifdef - define the forcePolicy to be cuda specific or not.